ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -190,37 +191,30 @@ then
 	set FMU_MODE pwm
 	set AUX_MODE pwm
 	set FMU_ARGS ""
-	set MAVLINK_F default
-	set MAVLINK_COMPANION_DEVICE /dev/ttyS2
```

Comment:
I think we still need to set this initially so that specific airframes and boards can override.

Reply:
Agreed.  Thanks.  I'll add it asap.